hazelcast/src/main/java/com/hazelcast/monitor/MemberState.java

```diff
@@ -31,6 +31,21 @@
 
     String getAddress();
 
+    /**
```

Comment:
Can't wait till we simplify this to a key/value set :) No more explicit methods needed.

Reply:
Me too :) Thanks for the review.